### PR TITLE
feat/#28

### PR DESCRIPTION
**Pull request by Luiginicoletti**
Issue: 

<details open> 
  <summary>
    <b>Home Page initial grid layout</b>
  </summary>

In this pr an initial homepage layout was created based on the given layout
</details>

<details open> 
  <summary>
    <b>Bugfix</b>
  </summary>

- **Description**
N/A

- **Cause**
N/A

- **Solution**
N/A
</details>

<details> 
  <summary>
    <b>Changelog</b>
  </summary>
N/A
</details>

<details open> 
  <summary>
    <b>Visual evidences :framed_picture:</b>
I put a responsivity on switches grid about minWidth and maxWidth

_fullscreen_
image
_minimized_
image

  </summary>

</details>

<details open> 
  <summary>
    <b>Checklist</b>
  </summary>

  - [X] Issue linked
  - [x] Build working correctly
  - [ ] Tests created
</details>

<details> 
  <summary>
    <b>Additional info</b>
  </summary>
N/A
</details>

### CODE REVIEW

File: src/App.module.scss [hunk 1]
```diff
@@ -1,3 +1,52 @@
...
+  box-sizing: border-box;
+}
+
+.mainContainer {
+  width: 100vw;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+  border: 4px solid rgb(21, 233, 169);
+  padding: 4px;
```
Comment (aalmeida00) [inline src/App.module.scss:21]: Transforma px em rem.

File: src/App.module.scss [hunk 2]
```diff
@@ -1,3 +1,52 @@
...
+  grid-area: input;
+  border: 4px solid blue;
+  
+}
+
+.gridTabs {
+  grid-area: tabs;
+  border: 4px solid yellow;
+  
+}
```
Comment (aalmeida00) [inline src/App.module.scss:52]: eof

**Review by aalmeida00 — CHANGES_REQUESTED** [review]

File: src/App.module.scss [hunk 3]
```diff
@@ -1,3 +1,52 @@
-.teste {
-  background-color: rgb(244, 129, 129);
+body, h1, h2, h3, h4, h5, h6, p, blockquote, pre, dl, dd, ol, ul, figure, hr {
```
Comment (Alecell) [inline src/App.module.scss:1]: Pq isso ainda existe? 
image

Vamos usar o [normalize css](https://necolas.github.io/normalize.css/) pra resetar os roles

File: src/App.module.scss [hunk 4]
```diff
@@ -1,3 +1,52 @@
...
+.gridInput {
+  grid-area: input;
+  border: 4px solid blue;
+  
+}
+
+.gridTabs {
+  grid-area: tabs;
+  border: 4px solid yellow;
+
```
Comment (Alecell) [inline src/App.module.scss:51]: ajustar a identação

**Review by Alecell — CHANGES_REQUESTED** [review]

**Review by aalmeida00 — APPROVED** [review]
lgtm

File: src/App.module.scss [hunk 5]
```diff
@@ -1,3 +1,49 @@
...
+body {
+  font-family: Arial, sans-serif;
+}
+
+*,
+::before,
+::after {
+  box-sizing: border-box;
+}
```
Comment (Alecell) [inline src/App.module.scss:13]: Remover

**Review by Alecell — CHANGES_REQUESTED** [review]

File: src/App.module.scss (hunk 5 above)
Comment (Luiginicoletti) [reply src/App.module.scss]: Feito!

**Review by aalmeida00 — APPROVED** [review]

File: src/App.module.scss [hunk 6]
```diff
@@ -1,3 +1,39 @@
 .teste {
   background-color: rgb(244, 123, 123);
 }
```
Comment (Alecell) [inline src/App.module.scss:3]: Apaga

**Review by Alecell — CHANGES_REQUESTED** [review]

File: src/App.tsx [hunk 7]
```diff
@@ -1,26 +1,15 @@
...
+function App() {
   return (
-    <div className={scss.teste} id="asdasdas" onClick={() => ali('oi')}>
-      <p>Hello World! SIM?</p> {role.sim} {sim}
-      <Testinho />
+    <div className={scss.mainContainer}>
+      <div className={scss.gridContainer}>
+        <div className={scss.gridSwitches}></div>
+        <div className={scss.gridInput}></div>
+        <div className={scss.gridTabs}></div>
```
Comment (Alecell) [inline src/App.tsx:11]: Ajusta isso tbm, coloca um self closed por enquanto

**Review by Alecell — CHANGES_REQUESTED** [review]

**Review by Alecell — APPROVED** [review]

**Pull request merged**